frontend/src/hooks/queries/studylog.ts

```diff
@@ -73,6 +76,32 @@ export const useDeleteStudylogMutation = () => {
   });
 };
 
+export const useFetchTempSavedStudylog = () => {
+  const { user } = useContext(UserContext);
+  const { username } = user;
+
+  return useQuery([QUERY_KEY.tempSavedStudylog, username], () => requestGetTempSavedStudylog(), {
```

Comment:
여기도 이렇게만 넣어줘도 될거같습니다!

```suggestion
  return useQuery([QUERY_KEY.tempSavedStudylog, username], requestGetTempSavedStudylog, {
```